ftw/zipextract/tests/test_zipextract.py

```diff
@@ -228,7 +229,7 @@ def test_zip_extraction_view_works(self, browser):
         browser.visit(self.file, view="zipextract")
         file_tree = browser.css(".zipextract.file_tree li")
         id_list = map(lambda el: el.node.get("id"), file_tree)
-        expected_ids = ['test', 'dir1', 'dir1-test2',
+        expected_ids = ['test', '-dir3', '-dir3-test5', 'dir1', 'dir1-test2',
```

Comment:
I don't really understand why -dir3 starts with -. I thought you stripped that.

Reply:
These are the path ID's an internal representation of the paths just for `ftw.zipextractor`

https://github.com/4teamwork/ftw.zipextract/blob/90ee96cce9c8f645471b710e78486bf07061c446/ftw/zipextract/zipextracter.py#L65-L74

https://github.com/4teamwork/ftw.zipextract/blob/90ee96cce9c8f645471b710e78486bf07061c446/ftw/zipextract/zipextracter.py#L29-L33

Reply:
Ah, okay thanks